### ISSUE TITLE
Default ssl on a custom domain for the admin panel

### ISSUE THREAD

**Issue by theRAGEhero** [jc21/nginx-proxy-manager#816]
Hello and thank you for your awesome job.

I was wondering if it's possible to have the admin panel on SSL just after the docker-compose installation.

I was thinking it in this way: I add a domain in the docker-compose file. That domain is going to be used for the admin panel (instead of port 81).

Once that the installation is complete, npm asks for an SSL certificate on that domain. 

If the domain is not written in the .yml file, then npm use the port 81 by default.

Maybe it's a stupid idea, but maybe no.

Thank you again!

**Comment by chaptergy** [comment]
I'll mark this as a duplicate of https://github.com/jc21/nginx-proxy-manager/issues/182, since it generally is. Your specific request to set the data in the docker-compose file could be a comment over there.
